piktorlog/src/components/molecules/MediaCard.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import {Button, Card, Image, Dropdown, DropdownItem, Modal, Popup, Divider} from 'semantic-ui-react';
-import { Redirect, Link } from 'react-router-dom';
+import {Button, Card, Image, Dropdown, /*DropdownItem,*/ Modal, Popup, /*Divider*/} from 'semantic-ui-react';
```

Comment:
Good catch here. We don't use the components you commented out. Will delete on my next PR.